autowebcompat/network.py

```diff
@@ -53,7 +53,7 @@ def create_vgg16_network(input_shape):
     x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)
 
     # Block 4
-    x = Conv2D(512, (3, 3), activation='relu', padding='same')(x)
+    x = Conv2D(512, (3, 3), flake8activation='relu', padding='same')(x)
```

Comment:
I suppose there's something wrong here :smile:

Reply:
:P Whoa! how did that happen